data_steward/resources/fields/note.json

```diff
@@ -25,12 +25,32 @@
         "mode": "required"
     }, 
     {
+        "type": "integer", 
+        "name": "note_class_concept_id", 
+        "mode": "required"
```

Comment:
Dumb question: how can we add required fields / make existing fields required? Won't old versions of OMOP data lack these?

(Can we force all data going forward to have these fields?)

Reply:
Good question. Short answer is the data is not directly portable up in that fashion. We can fill 0s as an interim solution, but it's not really the "right" solution.

We may end up with different OMOP versions over time in the CDR; we probably need a bigger picture answer to this as well. They try and keep new specs compatible with old code, but with a big version increment coming it may not be entirely true.

Reply:
(And as far as I am concerned, they don't have to be required; I think this is just following spec)